procmgr/processmanager.cpp

```diff
@@ -10098,8 +10076,11 @@ int ProcessManager::OAMParentModuleChange()
         status = startsystemthreadStatus;
     }
 
+    reinitProcessType("cpimport");
+
     // waiting until dml are ACTIVE
-    while (true)
+    int retry = 0;
+    while (retry < 30)
     {
         ProcessStatus DMLprocessstatus;
```

Comment:
In this and all of the new wait-for-dml blocks, what is ProcessOpState if/when getProcessStatus() throws an exception?